### ISSUE TITLE
Trust banner: clickable things within not tab-accessible

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#125097]
I can click into the banner and get a blue focus ring indicating keyboard focus but I cannot tab into the links or the close action inside.

![image](https://user-images.githubusercontent.com/900690/120334011-8d178300-c2f0-11eb-88a6-84809adb6a37.png)

//cc @isidorn

**Comment by isidorn** [comment]
This is as designed. You can click and use left / right to navigate across actions. Just like the status bar does today. ANd how our other toolbars do.
The reason for this is that we do not want the banner to occupy too many tab stops.

I leave it up to @lszomoru to close this.
But I just double checked this and I think it behaves fine.

**Comment by isidorn** [comment]
More details https://github.com/microsoft/vscode/issues/124165

**Comment by lszomoru** [comment]
Thanks @isidorn for sharing context. Closing this for now.
